lib/geordi/commands/drop_databases.rb

```diff
@@ -42,9 +47,11 @@ def drop_databases
     postgres_flags = "--port=#{options.postgres}"
   end
 
-  extra_flags = { 'mysql' => mysql_flags,
-                 'postgres' => postgres_flags }
-  cleaner = DBCleaner.new(extra_flags)
+  extra_flags = {
```

Comment:
I'd print the assumption above this line:

```
Interaction.note 'Assuming standard makandra setup. Run with --no-default if this does not work for you.'
```

Reply:
I don't think it makes sense here. It would also be displayed, if you already ran the command with the `no-default` option. I think I'd leave it in the initializer in db_cleaner.rb, or at least put this in an if-block here

Reply:
We decided to keep it here with a condition `if options.sudo`